Pack.tests.template.ps1

```diff
@@ -33,18 +32,21 @@ foreach ($packFile in $packFiles) {
 
             It "Has a details.json" {
                 $packFile.FullName | Should Exist
+                $packFile.Name | Should BeExactly "details.json"
```

Comment:
In VSCode only the test name is surfaced - can you either update it to include this restriction or (preferably) create another test?  Good unit testing practise is to only test a single thing with each test.

Reply:
To increase the readability of the output I'll change the test name to "Has a details.json (case-sensitive)", this should help a user that gets the error even through they have a details.JSON in the folder.